index.html

```diff
@@ -15,8 +15,8 @@ <h2 class="title-navbar">Developer Tools</h2>
   </nav>
   <div class="container">
     <h2 class="title">Front-End Tools</h2>
-    <div class="display-wrap">
-      <div class="tech-panel">
+    <div class="grid">
```

Comment:
No need to change the class name, remember to always use self explanatory class names